README.md

```diff
@@ -0,0 +1,29 @@
+## lcs
+Landlords checking service (lcs) form built using the HOF (Home Office Forms) Framework.
+
+## Getting Started
+### Prerequisites
+- [Node.js](https://nodejs.org/en/) - Tested against LTS
+- NPM (installed with Node.js) - Works with version 18+
+- [Html-pdf-converter](https://github.com/UKHomeOffice/html-pdf-converter) service running on a separate port to this service
+- [Redis server](http://redis.io/download) running on the default port
+
```

Comment:
maybe delete all of the below in the ReadME as that is more for a dev to write about env variables and I'm not sure what is needed